### DISCUSSION TITLE
Browsing by label / displaying catalog number

### DISCUSSION THREAD

**Discussion by Hydronicx** [navidrome/navidrome#2883] [Q&A]
Is there currently a way to browse by record label? Or are there any plans to implement this in the future?
This would be very useful for those record labels that specialize in compilations for instance.
Displaying the record label and catalog number on an album would be very nice (fetched from tags).
From the view of the album, it would be nice to browse to the music label and see other releases from the same label.
Perhaps this could be configured to be turned ON/OFF in case other people wouldn't like to see this info. Or perhaps only display it if the tags are present.

**Comment by crispybegs** [comment]
would love this pls

**Comment by coltercarson** [comment]
Agree this would be super useful - Even a label view added to the left side navigation menu would be handy (where Artist and Album are currently)
